lib/ansible/modules/cloud/vmware/vmware_guest.py

```diff
@@ -2709,7 +2710,7 @@ def get_vm_events(self, vm, eventTypeIdList):
         eventManager = self.content.eventManager
         return eventManager.QueryEvent(filterSpec)
 
-    def wait_for_customization(self, vm, poll=10000, sleep=10):
+    def wait_for_customization(self, vm, poll=360, sleep=10):
```

Comment:
Can we add a parameter like `wait_for_customization_timeout` just like `wait_for_ip_address_timeout` ?

Reply:
Ok, I think so. Thanks.

Reply:
Add new parameter `wait_for_customization_timeout` for this. Thanks.